docs/contributing/new-ruby-version.md

```diff
@@ -0,0 +1,23 @@
+# Adding support for new version of Ruby
+ 
+Following files needs to be updated to add support for a new version of already supported Ruby interpreter.
```

Comment:
I think this should start "The following files" and then "of an already-supported".